public/css/main.scss

```diff
@@ -16,6 +16,17 @@
   }
 }
 
+// Imgur Button CSS
+.btn-imgur {
+  background-color: #1bb76e;
+  color: #fff !important;
+
+  &:hover,
+  &:active {
+    background-color: #158c56;
+  }
+}
+
```

Comment:
The only button should be in api/index.pug
We have been trying not to add CSS for those buttons in main.scss, and just set the colors in the template.